jax/_src/numpy/lax_numpy.py

```diff
@@ -5973,19 +5973,44 @@ def meshgrid(*xi: ArrayLike, copy: bool = True, sparse: bool = False,
   return output
 
 
-@custom_jvp
-@util.implements(np.i0)
 @jit
 def i0(x: ArrayLike) -> Array:
+  """Calculate modified Bessel function of first kind, zeroth order.
+
+  JAX implementation of :func:`numpy.i0`.
```

Comment:
It might be nice to add the mathematical formula here (you can copy it from jax.scipy's `i0` docstring).

Reply:
Done